### PR TITLE
check app_code in bk-permission plugin

### PR DESCRIPTION
**Pull request by alex-smile**
- 修复 bk-permission 插件中，app_code 可能为空的问题

### CODE REVIEW

File: src/apisix/plugins/bk-permission.lua [hunk 1]
```diff
@@ -72,31 +71,34 @@ function _M.access(conf, ctx)
         return
     end
 
+    local app_code = ctx.var.bk_app_code
+
+    if pl_types.is_empty(app_code) then
+        return errorx.exit_with_apigw_err(
+            ctx, errorx.new_app_no_permission():with_fields(
+                {
+                    reason = "no permission, app is not verified, please provide valid app info to verify app.",
```
Comment (wklken) [inline src/apisix/plugins/bk-permission.lua:80]: please contact the administrator of this gateway. The reason for the error here is that: if resource_perm_required is true, verified_app_required must be set to true too.

File: src/apisix/tests/test-bk-permission.lua [hunk 2]
```diff
@@ -119,13 +122,31 @@ describe(
...
+                    cache_fallback, "get_with_fallback", function()
+                        return {}, nil
+                    end
+                )
+                ctx.var.bk_resource_auth.resource_perm_required = true
+                ctx.var.bk_app_code = ""
+                plugin.init()
+                local code = plugin.access(conf, ctx)
+                assert.is_not_nil(code)
+                assert.stub(cache_fallback.get_with_fallback).was_called(0)
```
Comment (wklken) [inline src/apisix/tests/test-bk-permission.lua:137]: NIT: code和error message assert一下

**Review by Han-Ya-Jun — APPROVED** [review]
LGTM

**Review by wklken — APPROVED** [review]
LGTM

**Review by zhu327 — APPROVED** [review]

**Pull request merged**